updatehub/tests/integration_test.rs

```diff
@@ -0,0 +1,270 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
+
+fn write_basic_config(path: PathBuf) -> String {
+    format!(
```

Comment:
You can use `toml::toml!` macro to allow for a nicer format of it. See how we did in:
https://github.com/OSSystems/EasySplash/blob/master/src/animation.rs#L147-L158